src/urllib3/util/request.py

```diff
@@ -103,6 +103,20 @@ def make_headers(
     return headers
 
 
+@overload
+def set_file_position(
+    body: IO[Any], pos: Optional[Union[int, object]]
+) -> Optional[Union[int, object]]:
+    ...
+
+
+@overload
+def set_file_position(
```

Comment:
Hmm, we might have to remove the `@overload` here and just go with `body: Any, pos: Optional[Union[int, object]] -> Optional[Union[int, object]]`